charts/nats/templates/jetstream-job-scc.yaml

```diff
@@ -0,0 +1,41 @@
+###################################
+### Astronomer Jetstrem Job SCC ###
+###################################
+{{- if .Values.global.sccEnabled }}
+{{- if .Values.nats.createJetStreamJob -}}
+{{- if or .Values.nats.jetstream.enabled .Values.global.nats.jetStream.enabled .Values.global.stan.enabled }}
```

Comment:
There are at least 6 possibilities of how these can be configured that have an impact on the code path. Currently there is only 1 test case. We should improve that.